opslevel/resource_opslevel_user.go

```diff
@@ -32,21 +32,19 @@ type UserResource struct {
 
 // UserResourceModel describes the User managed resource.
 type UserResourceModel struct {
-	Email            types.String `tfsdk:"email"`
-	Id               types.String `tfsdk:"id"`
-	LastUpdated      types.String `tfsdk:"last_updated"`
-	Name             types.String `tfsdk:"name"`
-	Role             types.String `tfsdk:"role"`
-	SkipWelcomeEmail types.Bool   `tfsdk:"skip_welcome_email"` // not usable but kept for backwards compatibility
+	Email       types.String `tfsdk:"email"`
+	Id          types.String `tfsdk:"id"`
+	LastUpdated types.String `tfsdk:"last_updated"`
+	Name        types.String `tfsdk:"name"`
+	Role        types.String `tfsdk:"role"`
 }
 
 func NewUserResourceModel(user opslevel.User) UserResourceModel {
 	return UserResourceModel{
-		Email:            RequiredStringValue(user.Email),
-		Id:               ComputedStringValue(string(user.Id)),
-		Name:             RequiredStringValue(user.Name),
-		Role:             OptionalStringValue(string(user.Role)),
-		SkipWelcomeEmail: types.BoolNull(),
```

Comment:
Its unclear to me why we did this.  This feature was requested by customers and should be configurable.

Reply:
I don't remember either... but it's back!